selectors/bacom-blog/sharepoint.page.js

```diff
@@ -11,39 +11,87 @@ export default class Sharepoint {
     this.insertPageBreakButton = this.iframe.locator('#InsertPageBreak');
     this.homeButton = this.iframe.locator('#Home');
     this.undoButton = this.iframe.locator('#UndoRedo > button:first-of-type');
+    this.editingElement = this.iframe.locator('#WACViewPanel_EditingElement');
+    this.documentTitle = this.iframe.locator('button#documentTitle');
+    this.dialogPanel = this.iframe.locator('#WACDialogPanel');
+    this.dialogText = this.iframe.locator('#WACDialogTextPanel');
+    this.navButton = this.iframe.locator('#FishBowlNavButton');
+    this.pageBreaks = this.iframe.locator('span.PageBreakTextSpan');
   }
 
   /**
-   * @description Adds a page break by selecting the insert page break button under
-   * the inserts tab and then saves the file.
+   * @description Gets the text of the dialog box.
+   * @returns {Promise<string>} The text of the dialog box.
+   */
+  async getDialogText() {
+    if (await this.dialogText.isVisible()) {
+      return this.dialogText.textContent();
```

Comment:
you need await for this

Reply:
I mean for textContent()

Reply:
ESLint will complain of redundant use of `await` on a return. (https://eslint.org/docs/rules/no-return-await)
I can change it to this:
```suggestion
      const dialogText = await this.dialogText.textContent();
      return dialogText;
```